private/src/styles/base/_type.scss

```diff
@@ -26,31 +10,10 @@ p.has-drop-cap:not(:focus)::first-letter {
   margin-bottom: 30px;
 }
 
-p.has-small-font-size {
-  font-size: .9rem;
-}
-
-.has-normal-font-size,
-p.has-regular-font-size {
-  font-size: $base-font-size;
-}
-
-p.has-large-font-size {
-  font-size: 1.5rem;
-}
-
-p.has-larger-font-size {
-  font-size: 2rem;
-}
-
-.has-x-large-font-size {
-  font-size: 2.25em;
-}
-
-.has-huge-font-size {
-  font-size: 2.5rem;
+small {
+  font-size: var(--wp--preset--font-size--small);
 }
 
-small {
-  font-size: $milli-size;
+.has-normal-font-size {
```

Comment:
I left this `has` class in, as I went with `regular` as the name of this font size, so this just ensures this class and the new variable are synced.